solidity/contracts/ECDSARewards.sol

```diff
@@ -100,6 +116,33 @@ contract ECDSARewards is Rewards {
         )
     {
         factory = BondedECDSAKeepFactory(_factoryAddress);
+        tokenStaking = TokenStaking(_tokenStakingAddress);
+    }
+
+    /// @notice Withdraw all available rewards for the given interval.
+    /// The rewards will be paid to the beneficiary
+    /// of the operator requesting the withdrawal.
+    /// @param interval The interval
+    function withdrawRewards(uint256 interval) external {
+        address beneficiary = tokenStaking.beneficiaryOf(msg.sender);
```

Comment:
If we do it this way, only operator can withdraw. I think we should rather allow to pass the operator/beneficiary as a parameter.

Reply:
Operator added as parameter; we can also switch to beneficiary if we like that better.

Reply:
I think the operator makes more sense 👌